firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTests.swift

```diff
@@ -46,6 +50,30 @@ class TabTests: XCTestCase {
         XCTAssertEqual(tab.displayTitle, expectedDisplayTitle)
     }
 
+    func testTitle_WhenWebViewTitleIsNil_ShouldReturnNil() {
```

Comment:
very small nit; we can add `Then` to specify this is the expected result (following the Given-When-Then naming style)
```suggestion
    func testTitle_WhenWebViewTitleIsNil_ThenShouldReturnNil() {
```